lib/rollup-build.js

```diff
@@ -0,0 +1,46 @@
+const { resolve } = require('path')
```

Comment:
Do we want to add test fixtures for this (aping things like `extract-page-headings`, `get-macro-options` and `metalsmith-lunr-index`)?

It'd be nice to catch anything weird without having to rely on diffing the dist folder, but a bit onerous to get a decent suite going (though I guess we could take the lead from [`metalsmith-rollup`](https://github.com/attila/metalsmith-rollup/tree/master/test)).